src/components/tile-select/tile-select.style.js

```diff
@@ -58,15 +55,34 @@ const StyledTileSelect = styled.div`
         fill: ${theme.colors.black};
         opacity: 0.3;
       }
-    `};
+    `}
+`;
+
+const StyledFocusWrapper = styled.div`
+  position: relative;
+  border: 1px solid ${({ theme }) => theme.tileSelect.border};
+
+  ${({ checked, theme }) =>
+    checked &&
+    css`
+      border-color: ${theme.colors.primary};
+      z-index: 10;
+    `}
+
+  ${({ hasFocus }) =>
+    hasFocus &&
+    css`
+      outline: 3px solid ${({ theme }) => theme.colors.focus};
+      z-index: 15;
+    `}
```

Comment:
Hey! I wonder we can put all props at the begging of the `StyledFocusWrapper` like
```js
const StyledFocusWrapper = styled.div`
${({checked, theme, hasFocus}) && css``
`
```